patches/chrome-test-base-chrome_unit_test_suite.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/test/base/chrome_unit_test_suite.cc b/chrome/test/base/chrome_unit_test_suite.cc
+index 1f940289d0c4b301385aa7c2382be4209656ff6d..8bcbc393e6b527f509a3c33aa9a2f2611e7b1293 100644
+--- a/chrome/test/base/chrome_unit_test_suite.cc
++++ b/chrome/test/base/chrome_unit_test_suite.cc
+@@ -79,6 +79,7 @@ class ChromeUnitTestSuiteInitializer : public testing::EmptyTestEventListener {
+     content::SetUtilityClientForTesting(utility_content_client_.get());
+ 
+     TestingBrowserProcess::CreateInstance();
++    TestingBraveBrowserProcess::CreateInstance();
```

Comment:
Would we have any undesirable side effects if we subclassed `TestingBrowserProcess` and called `TestingBraveBrowserProcess::[Create|Delete]Instance` in the overrides of the corresponding statics? I haven't tried but I think that would let us avoid patching.

Reply:
I got that working in https://github.com/brave/brave-core/pull/8707/commits/a979f8ca266b7532aece2f70c20a86fea6e86f7f, apart from some new `gn_check` issues. Do you know how to fix these?

```
ERROR in /home/antonok/brave/recent/brave-browser/src/brave/chromium_src/chrome/test/base/testing_browser_process.cc
  Illegal include: "../../../../chrome/test/base/testing_browser_process.cc"
    Because of no rule applying.

ERROR in /home/antonok/brave/recent/brave-browser/src/brave/chromium_src/chrome/test/base/testing_browser_process.h
  Illegal include: "../../../../chrome/test/base/testing_browser_process.h"
    Because of no rule applying.
```

Reply:
weirdly enough, `"+../../../../../chrome/test/base"` was already in there, the issue was that I missed an extra `../` segment in the includes. I'm surprised that compiled at all! Should be fixed now though.

Reply:
Yeah, DEPS doesn't affect compilation, only affects gn_check.